Obsidian/Registries/CommandsRegistry.cs

```diff
@@ -27,9 +27,9 @@ public static void Register(Server server)
                 Type = CommandNodeType.Literal
             };
 
-            foreach (var overload in cmd.Overloads.Take(1))
+            foreach (var overload in cmd.Overloads)
             {
-                var args = overload.GetParameters().Skip(1); // skipping obsidian context
+                var args = overload.GetParameters();
```

Comment:
This may or may not break docker support, I believe this is why I added it. Might not be relevant anymore. I don't remember.

Reply:
Take a look at [CommandExecutor.cs & CommandExecutor.Delegate.cs](https://github.com/ObsidianMC/Obsidian/tree/plugin-manager-cleanup/Obsidian/Commands/Framework)

Reply:
I'll take it.